src/thor/bidirectional_astar.cc

```diff
@@ -318,8 +318,8 @@ inline bool BidirectionalAStar::ExpandInner(baldr::GraphReader& graphreader,
 
   // setting this edge as reached
   if (expansion_callback_) {
-    expansion_callback_(graphreader, "bidirectional_astar", FORWARD ? meta.edge_id : opp_edge_id, "r",
-                        false);
+    expansion_callback_(graphreader, FORWARD ? meta.edge_id : opp_edge_id, "bidirectional_astar", "r",
+                        0, 0, 0);
```

Comment:
or you want duration, distance, cost for bdira* as well?

Reply:
why not, its incredibly useful debug info right?